applications/launch/proxy.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="urlServer" />
```

Comment:
@pablisho could you please just use 'url' as a param? Or if you choose a different name, please use this_fromat, not lowerCamelCase. Thanks!